qtpy/__init__.py

```diff
@@ -264,8 +264,11 @@ def __init__(self, *, missing_package=None, **superclass_kwargs):
 # If a correct API name is passed to QT_API and it could not be found,
 # switches to another and informs through the warning
 if API != initial_api and binding_specified:
-    warnings.warn('Selected binding "{}" could not be found, '
-                  'using "{}"'.format(initial_api, API), RuntimeWarning)
+    warnings.warn(
+        f'Selected binding {initial_api!r} could not be found; '
+        f'falling back to {API!r}',
+        PythonQtWarning,
+        )
```

Comment:
```suggestion
    )
```

Reply:
Rebased in; I was just using the Spyder default style rather than this Black style since we were going to run Black anyway and I thought other code here used it, but sure :+1: